plugins/wildfly_logs.yaml

```
@@ -0,0 +1,103 @@
+version: 0.0.1
+title: Wildfly
+description: Log parser for Wildfly
+parameters:
+  - name: standalone_file_path
+    type: "[]string"
+    default:  
+      - "/opt/wildfly/standalone/log/server.log"
+  - name: enable_domain_server
+    type: bool
+    default: false
+  - name: domain_server_path
+    type: "[]string"
+    default: 
+      - "/opt/wildfly/domain/servers/*/log/server.log"
+  - name: enable_domain_controller
+    type: bool
+    default: false      
+  - name: domain_controller_path
+    type: "[]string"
+    default: 
+      - "/opt/wildfly/domain/log/*.log"        
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+
+template: |
+  receivers:
+    filelog/standalone:
+      include:
+        {{ range $i, $fp := .standalone_file_path  }}
+          - '{{ $fp }}'
+        {{ end }}
+      start_at: {{ .start_at }}
+      multiline:
+        line_start_pattern: '\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}'
+      attributes:
+        log_type: wildfly
```

This is pretty nitpicky, but I feel like this shouldn't be just `wildfly`, since `wildfly` is more of a namespace.
```suggestion
        log_type: wildfly.standalone
```